src/ngraph/runtime/cache.cpp

```diff
@@ -0,0 +1,134 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/runtime/cache.hpp"
+
+using namespace ngraph;
+using namespace std;
+
+// Constructor
+runtime::LRUCache::LRUCache()
+{
+    char* cache_size = getenv("NGRAPH_CACHE_SIZE");
```

Comment:
Can we have an API to enable bridge to set the cache size through bridge? The FW can be restrictive in env var use.

Reply:
@gaurides is working on this.